### PR TITLE
Improves, standardizes pen labeling

### PR DESCRIPTION
**Pull request by 9600bauds**
Oh yeah and now you can label pill bottles with a pen.

### CODE REVIEW

File: code/modules/paperwork/handlabeler.dm [hunk 1]
```diff
@@ -116,36 +109,63 @@
...
+ */
+
+/atom/proc/set_labeled(var/label, var/start_text = " (", var/end_text = ")")
+	if(labeled)
+		remove_label()
+	labeled = "[start_text][label][end_text]"
+	name = "[name][labeled]"
+	new/atom/proc/remove_label_verb(src)
+
+/atom/proc/remove_label_verb()
```
Comment (PJB3005) [inline code/modules/paperwork/handlabeler.dm:142]: Needs sanity for whether the user's alive and such.

**Review by PJB3005 — DISMISSED** [review]

File: code/modules/paperwork/handlabeler.dm (hunk 1 above)
Comment (9600bauds) [reply code/modules/paperwork/handlabeler.dm:142]: Hahaha wonderful

**Comment by Probe1** [comment]
Don't dismiss others non meme review please.  I was going to merge this 12 hours ago but don't know if @PJB3005 is on board.  It makes my life a little harder.

**Comment by PJB3005** [comment]
Considering that was my only critique of the PR it's fine.

**Comment by 9600bauds** [comment]
But I specifically dismissed it as "added"

**Pull request merged**